server/pension/models.py

```diff
@@ -34,30 +115,28 @@ def __str__(self):
         return '{} / {}'.format(self.quarter, self.year)
 
 
-class ManagingBody(models.Model):
-    """
-    The body that owns the funds.
-    """
+class Fund(models.Model):
     label = models.CharField(max_length=200)
 
     def __str__(self):
         return self.label
 
-
-class Fund(models.Model):
+class Issuer(models.Model):
+    """
+    Company (or other body) that issues a security.
+    """
     label = models.CharField(max_length=200)
 
     def __str__(self):
         return self.label
 
-
 class FundManagingBody(models.Model):
     """
     A fund can move from one ManagingBody to another, this model documents the
     moves.
     """
     fund = models.ForeignKey(Fund)
-    managing_body = models.ForeignKey(ManagingBody)
+    managing_body = models.ForeignKey(Issuer)
```

Comment:
@ranmoshe From this `ForeignKey` I understand that now the `Issuer` is our `ManagingBody`, so the `views` and `serializer` make sense. But the `docs` here needs to be fixed (it cannot move from `one ManagingBody to another` because `ManagingBody` don't exist anymore.